src/agents/fundamentals.py

```diff
@@ -17,66 +17,87 @@ def fundamentals_agent(state: AgentState):
     
     # 1. Profitability Analysis
     profitability_score = 0
-    if metrics["return_on_equity"] > 0.15:  # Strong ROE above 15%
+    if metrics["return_on_equity"] and metrics["return_on_equity"] > 0.15:  # Strong ROE above 15%
```

Comment:
Thanks Kit - can we just extract the `metrics["return_on_equity"]` into a variable (similar to `pe_ratio`) [below](https://github.com/virattt/ai-hedge-fund/pull/39/files#diff-03fb6c77b1b2aa743a02e87bcc02069316d9ded9bb11558575ec06b4c036881aR79)?